assets/components/atoms/icon/icon.twig

```diff
@@ -0,0 +1 @@
+<svg class="icon"><use xlink:href="#{{ icon ?: 'icon-reading-time' }}"></use></svg>
```

Comment:
`icon-reading-time` is not the right default icon (it's IMD's) please use an EPFL one 😉